tests/interfaces/test_torch.py

```diff
@@ -974,7 +974,7 @@ def test_sampling_gradient_error(self, torch_device, execute_kwargs):
 
         with qml.queuing.AnnotatedQueue() as q:
             qml.RX(x, wires=[0])
-            qml.sample(qml.PauliZ(0))
+            qml.sample()
```

Comment:
This is to keep sampling `int`s for this test?

Reply:
Yes! Alternatively, we can leave the `tape` as-is, and change the expected error message - after these changes, the original tape in this test fails with an error `'grad can be implicitly created only for scalar outputs'`. Neither of the errors feel super specific to this to me, so I'm not actually sure what this test is for, so I'm not sure how best to update it 🤔

Reply:
I think it's fine as is! :)